new/analyse/determine-version.ts

```diff
@@ -0,0 +1,53 @@
+import type { Module, PackageAfterDetermineVersion } from '../types';
+
+import { logger } from '../helpers/logger';
+import { clone } from 'ramda';
+import semver from 'semver';
+
+const { error, log } = logger('determine Version (1)');
+
+export const determineVersion: Module = (env) => {
+  if (!env.packages) {
+    const errorMessage = '"packages" not present on env config object.';
+    error(errorMessage);
+    throw new Error(errorMessage);
+  }
+
+  log('analyse packages');
+
+  const clonePackages = clone(env.packages);
+
+  const incrementLevels: semver.ReleaseType[] = ['patch', 'minor', 'major'];
+
+  Object.keys(clonePackages).forEach((packageName) => {
+    const { determinedIncrementLevel, packageJson } =
+      clonePackages[packageName];
+
+    const version: semver.SemVer | null = semver.parse(packageJson.version);
```

Comment:
I guess `package.json` does not contain the version number. (as it usually comes from `rlsr.json`)

Reply:
Seems like that part is still missing. We read the `rlsr.json`, but we don't assign its content to the packages

Reply:
We do that in a separate PR ... I've just added the doc comments for that.